programs/ziti-edge-tunnel/package/rpm/post.sh.in

```diff
@@ -1,8 +1,20 @@
 SYSTEMD_SERVICE_NAME=@SYSTEMD_SERVICE_NAME@
 SYSTEMD_UNIT_FILE_NAME=@SYSTEMD_UNIT_FILE_NAME@
-if [ $1 -eq 1 ]; then
```

Comment:
1 means install, but we need to run these commands for upgrades (2) too, excluding remove (0).